### PR TITLE
Travis

### PR DESCRIPTION
**Pull request by mattgibson09**
Merging to master to fix commit history

### CODE REVIEW

File: install-dependencies.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+#install client dependencies
+cd client
+npm install
+#install server dependencies
+cd ../server
+npm install
+#restore initial working directory
+cd ..
```
Comment (nashley) [inline install-dependencies.sh]: Add a newline for POSIX compliance or whatever.

Comment (nashley) [inline install-dependencies.sh]: Add spaces to beginning of comments in this file to match the `.travis.yml` file

File: install-dependencies.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+#install client dependencies
+cd client
+npm install
+#install server dependencies
+cd ../server
```
Comment (nashley) [inline install-dependencies.sh]: Might be better to reference this like `~/client` and `~/server`. Not super important.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+language: node_js
+install: ./install-dependencies.sh
+script:
+#- jest server/
```
Comment (nashley) [inline .travis.yml]: Why're these two lines commented out?

Comment (mattgibson09) [reply .travis.yml]: Placeholders for actual scripts that are now provided

**Pull request merged**